.github/workflows/native-cron-build.yml

```diff
@@ -39,34 +42,21 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
 
+      - name: Run dependency checker
+        run: |
+          echo "@AGG Inside of dependency checker task"
+          echo "GITHUB_SHA=${GITHUB_SHA}"
+          echo "GITHUB_REF=$GITHUB_REF"
+          echo "GITHUB_HEAD_REF=$GITHUB_HEAD_REF"
+          echo "GITHUB_BASE_REF=$GITHUB_BASE_REF"
+          mvn -B -f independent-projects/utilities/dependency-checker/pom.xml package
+          java -jar independent-projects/utilities/dependency-checker/target/quarkus-utilities-dependency-checker-999-SNAPSHOT-jar-with-dependencies.jar ${GITHUB_BASE_REF} ${GITHUB_HEAD_REF}
```

Comment:
This section is just a WIP while I figure out how to properly hook the tool into actions